tests/classes/test_function.py

```diff
@@ -0,0 +1,105 @@
+from attr import attr
+import pytest
+import xgi
+import numpy as np
+
+from xgi.classes.hypergraph import Hypergraph
+from xgi.exception import XGIError
+
+
+def test_degree_histogram(edgelist1, edgelist2, edgelist3):
+    H1 = xgi.Hypergraph(edgelist1)
+    H2 = xgi.Hypergraph(edgelist2)
+    H3 = xgi.Hypergraph(edgelist3)
+
+    assert xgi.degree_histogram(H1) == [0, 7, 1]
+    assert xgi.degree_histogram(H2) == [0, 5, 1]
+    assert xgi.degree_histogram(H3) == [0, 4, 2]
```

Comment:
Suggestion (non-blocking): perhaps in the future we could provide two different functions:
1. `degree_counts` returns what `degree_histogram` currently returns, and
2. `degree_histogram` returns the same data structures as [np.histogram](https://numpy.org/doc/stable/reference/generated/numpy.histogram.html) (i.e. one array with the heights and one array with the bin edges.)

Reply:
Added as an issue!